chia/wallet/wallet_state_manager.py

```diff
@@ -1431,7 +1473,7 @@ async def _add_coin_states(
                                         sent_to=[],
                                         trade_id=None,
                                         type=uint32(TransactionType.OUTGOING_TX.value),
-                                        name=bytes32(token_bytes()),
+                                        name=tx_name,
```

Comment:
Seems like a good idea to not use random bytes for a tx hash yeah but its probably not related to the actual problem here?

Reply:
This is the original fix for the duplicate issue since we saw duplicate tx records. It seems unrelated now, but good to have. Let me know if you think it's necessary to create another PR for this.

Reply:
Hm i will defer to @trepca here but for me it seems like we don't need this in the release then and we can PR it against main instead.

Reply:
@ytx1991  lets PR against main